bundles/org.openhab.binding.emby/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,195 @@
+	<channel-type id="title">
+		<item-type>String</item-type>
+		<label>Title</label>
+		<description>Title of the current song</description>
+		<state readOnly="true" pattern="%s" />
+	</channel-type>
+	<channel-type id="showtitle">
+		<item-type>String</item-type>
+		<label>Show Title</label>
+		<description>Title of the current show</description>
+		<state readOnly="true" pattern="%s" />
+	</channel-type>
+	<channel-type id="control">
+		<item-type>Player</item-type>
+		<label>Control</label>
+		<description>Control the Emby Player, e.g. start/stop/next/previous/ffward/rewind</description>
+		<category>Player</category>
+	</channel-type>
+	<channel-type id="stop">
+		<item-type>Switch</item-type>
+		<label>Stop</label>
+		<description>Stops the player. ON if the player is stopped.</description>
+	</channel-type>
+	<channel-type id="mute">
+		<item-type>Switch</item-type>
+		<label>Mute</label>
+		<description>Mute/unmute your device</description>
+	</channel-type>
+	<channel-type id="currenttime">
+		<item-type>Number:Time</item-type>
+		<label>Current Time</label>
+		<description>Current time of currently playing media</description>
+		<state readOnly="true" pattern="%d %unit%" />
+	</channel-type>
+	<channel-type id="duration">
+		<item-type>Number:Time</item-type>
+		<label>Duration</label>
+		<description>Length of currently playing media</description>
+		<state readOnly="true" pattern="%d %unit%" />
+	</channel-type>
+	<channel-type id="mediatype">
+		<item-type>String</item-type>
+		<label>Media Type</label>
+		<description>Media type of the current file</description>
+		<state readOnly="true" pattern="%s" />
+	</channel-type>
+	<channel-type id="generalCommand">
+		<item-type>Switch</item-type>
+		<label>General Command</label>
+		<description>Send a general command</description>
+		<config-description>
+			<parameter name="generalCommand_CommandName" type="text">
+				<options>
+					<option value="MoveUp">Move Up</option>
+					<option value="MoveDown">Move Down</option>
+					<option value="MoveLeft">Move Left</option>
+					<option value="MoveRight">Move Right</option>
+					<option value="PageUp">Page Up</option>
+					<option value="PageDown">Page Down</option>
+					<option value="PreviousLetter">Previous Letter</option>
+					<option value="NextLetter">Next Letter</option>
+					<option value="ToggleOsdMenu">Toggle Osd Menu</option>
+					<option value="ToggleContextMenu">Toggle Context Menu</option>
+					<option value="ToggleMute">Toggle Mute</option>
+					<option value="Select">Select</option>
+					<option value="Back">Back</option>
+					<option value="TakeScreenshot">Take Screenshot</option>
+					<option value="GoHome">Go Home</option>
+					<option value="GoToSettings">Go To Settings</option>
+					<option value="VolumeUp">Volume Up</option>
+					<option value="VolumeDown">Volume Down</option>
+					<option value="ToggleFullscreen">Toggle Full Screen</option>
+					<option value="GoToSearch">Go To Search</option>
```

Comment:
If I understand correctly, these would be rather Actions, too. Same for below. Then, you don't need to rely on the JSON code, too.

Reply:
You are correct this makes this much cleaner.  Will revise